libraries/kotlin.test/testng/src/main/kotlin/Annotations.kt

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright 2000-2018 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the license/LICENSE.txt file.
+ */
+
+
+@file:Suppress("INVISIBLE_MEMBER", "INVISIBLE_REFERENCE")
+@file:JvmPackageName("kotlin.test.annotations")
+package kotlin.test
+
+public actual typealias Test = org.testng.annotations.Test
+public actual typealias Ignore = org.testng.annotations.Ignore
```

Comment:
Interesting, I hadn't noticed this annotation in javadoc of testng before, seems that it's rather new. What is the testng version it's available from?

Reply:
I suppose you talking about `@Ignore`, i believe it was added in latest version 6.13